nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/resources/TestFilterCSVColumns/US500.csv

```diff
@@ -0,0 +1 @@
"Sherita","Saras","Black History Resource Center","8 Us Highway 22","Colorado Springs","El Paso","CO",80937,"719-669-1664","719-547-9543","sherita.saras@cox.net","http://www.blackhistoryresourcecenter.com"
"Lashawnda","Stuer","Rodriguez, J Christopher Esq","7422 Martin Ave #8","Toledo","Lucas","OH",43607,"419-588-8719","419-399-1744","lstuer@cox.net","http://www.rodriguezjchristopheresq.com"
"Ernest","Syrop","Grant Family Health Center","94 Chase Rd","Hyattsville","Prince Georges","MD",20785,"301-998-9644","301-257-4883","ernest@cox.net","http://www.grantfamilyhealthcenter.com"
"Nobuko","Halsey","Goeman Wood Products Inc","8139 I Hwy 10 #92","New Bedford","Bristol","MA","02745","508-855-9887","508-897-7916","nobuko.halsey@yahoo.com","http://www.goemanwoodproductsinc.com"
"Lavonna","Wolny","Linhares, Kenneth A Esq","5 Cabot Rd","Mc Lean","Fairfax","VA",22102,"703-483-1970","703-892-2914","lavonna.wolny@hotmail.com","http://www.linhareskennethaesq.com"
"Lashaunda","Lizama","Earnhardt Printing","3387 Ryan Dr","Hanover","Anne Arundel","MD",21076,"410-678-2473","410-912-6032","llizama@cox.net","http://www.earnhardtprinting.com"
"Mariann","Bilden","H P G Industrys Inc","3125 Packer Ave #9851","Austin","Travis","TX",78753,"512-223-4791","512-742-1149","mariann.bilden@aol.com","http://www.hpgindustrysinc.com"
"Helene","Rodenberger","Bailey Transportation Prod Inc","347 Chestnut St","Peoria","Maricopa","AZ",85381,"623-461-8551","623-426-4907","helene@aol.com","http://www.baileytransportationprodinc.com"
"Roselle","Estell","Mcglynn Bliss Pc","8116 Mount Vernon Ave","Bucyrus","Crawford","OH",44820,"419-571-5920","419-488-6648","roselle.estell@hotmail.com","http://www.mcglynnblisspc.com"
"Samira","Heintzman","Mutual Fish Co","8772 Old County Rd #5410","Kent","King","WA",98032,"206-311-4137","206-923-6042","sheintzman@hotmail.com","http://www.mutualfishco.com"
"Margart","Meisel","Yeates, Arthur L Aia","868 State St #38","Cincinnati","Hamilton","OH",45251,"513-617-2362","513-747-9603","margart_meisel@yahoo.com","http://www.yeatesarthurlaia.com"
"Kristofer","Bennick","Logan, Ronald J Esq","772 W River Dr","Bloomington","Monroe","IN",47404,"812-368-1511","812-442-8544","kristofer.bennick@yahoo.com","http://www.loganronaldjesq.com"
"Weldon","Acuff","Advantage Martgage Company","73 W Barstow Ave","Arlington Heights","Cook","IL",60004,"847-353-2156","847-613-5866","wacuff@gmail.com","http://www.advantagemartgagecompany.com"
"Shalon","Shadrick","Germer And Gertz Llp","61047 Mayfield Ave","Brooklyn","Kings","NY",11223,"718-232-2337","718-394-4974","shalon@cox.net","http://www.germerandgertzllp.com"
"Denise","Patak","Spence Law Offices","2139 Santa Rosa Ave","Orlando","Orange","FL",32801,"407-446-4358","407-808-3254","denise@patak.org","http://www.spencelawoffices.com"
"Louvenia","Beech","John Ortiz Nts Therapy Center","598 43rd St","Be
```

Comment:
Do we know where this test data came from? We need to be sure to account for licensing information in a header.